test/with_dummyserver/test_chunked_transfer.py

```diff
@@ -66,11 +67,12 @@ def test_bytestring_body(self):
     def test_unicode_body(self):
         # Define u'thisshouldbeonechunk\r\näöüß' in a way, so that python3.1
```

Comment:
More of a though/optional comment: Since we no longer support 3.1, and are using unicode strings in other tests, is there a reason not to define this as `u'thisshouldbeonechunk\r\näöüß'`. It's a bigger surface area change on the test but seems more explicit in what we're trying to accomplish here.